functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/Project.java

```diff
@@ -36,6 +42,32 @@ public ProjectConfigurator configureSmartTesting() {
         return new ChangeApplier(repository).applyAsCommits(changeDescriptions);
     }
 
+    Project checkoutBranch(String branchName) {
+        try {
+            git.checkout().setName(branchName).call();
```

Comment:
I am not sure if this really should belong here. Maybe instead you can simply `.applyAsCommit("Disable surefire and enable just failsafe plugin")` like we do with other changes?